app/lib/controlled_work_mappings/civ_means_7_new.yml

```diff
@@ -0,0 +1,457 @@
+# This file defines the mappings used to prepopulate a CIVMEANS7 PDF form with the answers given by a user to CCQ's questions
+# combined with the output from CFE.
+
+# The mappings conform to the format described in ControlledWorkDocumentValueMappingService
+
+- section: general
+  fields:
+    # "The client is a child under the age of 18:"
+    - name: Client is under 18
+      type: boolean_radio
+      source: under_eighteen?
+      yes_value: "Yes"
+      no_value: "No"
+
+    - name: Application for controlled legal representation
+      type: boolean_radio
+      source: controlled_legal_representation
+      yes_value: "Yes"
+      no_value: "No"
+
+    # "The child’s means are to be aggregated with a maintaining adult."
+    - name: Means to be aggregated with maintaining adult
+      type: boolean_radio
+      source: aggregated_means
+      yes_value: "Yes"
+      no_value: "No"
+
+    # "Does this child receive any money on a regular basis? Do not include any part-time earnings"
+    - name: Child receives money
+      type: boolean_radio
+      source: regular_income
+      yes_value: "Yes"
+      no_value: "No"
+
+    # "Does this child have any savings, items of value or investments totalling £2,500 or more"
+    - name: Child has savings/items of value/investments totalling £2,500 or more
+      type: boolean_radio
+      source: under_eighteen_assets
+      yes_value: "Yes"
+      no_value: "No"
+
+    # "The client has a partner whose means are to be aggregated"
+    - name: Partner
+      type: boolean_radio
+      source: partner
+      yes_value: "Yes"
+      no_value: "No"
+
+    # "The client's assets (held in sole name or jointly held) have been claimed by the opponent"
+    - name: Assets claimed by opponent
+      type: boolean_radio
+      source: smod_assets?
+      yes_value: "Yes"
+      no_value: "No"
+- section: smod_capital
```

Comment:
maybe add a space between the two sections for readability like you have for the other sections